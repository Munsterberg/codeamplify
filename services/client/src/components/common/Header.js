import React from 'react';
import { Navbar, Nav, NavItem } from 'react-bootstrap';
import { LinkContainer } from 'react-router-bootstrap';

const Header = () => (
  <Navbar inverse collapseOnSelect>
    <Navbar.Header>
      <Navbar.Brand>
        <span>Brand</span>
      </Navbar.Brand>
      <Navbar.Toggle />
    </Navbar.Header>
    <Navbar.Collapse>
      <Nav>
        <LinkContainer to="/">
          <NavItem eventKey={1}>Home</NavItem>
        </LinkContainer>
        <LinkContainer to="/about">
          <NavItem eventKey={2}>About</NavItem>
        </LinkContainer>
      </Nav>
      <Nav pullRight>
        <LinkContainer to="/register">
          <NavItem eventKey={1}>Register</NavItem>
        </LinkContainer>
        <LinkContainer to="/login">
          <NavItem eventKey={2}>Login</NavItem>
        </LinkContainer>
      </Nav>
    </Navbar.Collapse>
  </Navbar>
);

export default Header;