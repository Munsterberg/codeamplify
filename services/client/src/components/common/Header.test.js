import React from 'react';
import { shallow } from 'enzyme';

import Header from './Header';

describe('<Header />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Header />);
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});