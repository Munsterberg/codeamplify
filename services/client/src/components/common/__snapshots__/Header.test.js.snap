// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render correctly 1`] = `
<Uncontrolled(Navbar)
  bsStyle="default"
  collapseOnSelect={true}
  inverse={true}
>
  <NavbarHeader>
    <NavbarBrand>
      <span>
        Brand
      </span>
    </NavbarBrand>
    <NavbarToggle />
  </NavbarHeader>
  <NavbarCollapse>
    <Nav
      bsClass="nav"
      justified={false}
      pullLeft={false}
      pullRight={false}
      stacked={false}
    >
      <LinkContainer
        activeClassName="active"
        exact={false}
        replace={false}
        strict={false}
        to="/"
      >
        <NavItem
          active={false}
          disabled={false}
          eventKey={1}
        >
          Home
        </NavItem>
      </LinkContainer>
      <LinkContainer
        activeClassName="active"
        exact={false}
        replace={false}
        strict={false}
        to="/about"
      >
        <NavItem
          active={false}
          disabled={false}
          eventKey={2}
        >
          About
        </NavItem>
      </LinkContainer>
    </Nav>
    <Nav
      bsClass="nav"
      justified={false}
      pullLeft={false}
      pullRight={true}
      stacked={false}
    >
      <LinkContainer
        activeClassName="active"
        exact={false}
        replace={false}
        strict={false}
        to="/register"
      >
        <NavItem
          active={false}
          disabled={false}
          eventKey={1}
        >
          Register
        </NavItem>
      </LinkContainer>
      <LinkContainer
        activeClassName="active"
        exact={false}
        replace={false}
        strict={false}
        to="/login"
      >
        <NavItem
          active={false}
          disabled={false}
          eventKey={2}
        >
          Login
        </NavItem>
      </LinkContainer>
    </Nav>
  </NavbarCollapse>
</Uncontrolled(Navbar)>
`;
